packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -224,6 +224,74 @@ void main() {
         expect(findState(const Key('A')).checked, true);
       });
 
+      testWidgets('Uses the PrimaryScrollController when available', (WidgetTester tester) async {
+        final ScrollController primary = new ScrollController();
+        final Widget reorderableList = new ReorderableListView(
+          children: const <Widget>[
+            SizedBox(width: 100.0, height: 100.0, child: Text('C'), key: Key('C')),
+            SizedBox(width: 100.0, height: 100.0, child: Text('B'), key: Key('B')),
+            SizedBox(width: 100.0, height: 100.0, child: Text('A'), key: Key('A')),
+          ],
+          onReorder: (int oldIndex, int newIndex) {},
+        );
+
+        Widget buildWithScrollController(ScrollController controller) {
+          return new MaterialApp(
+            home: new PrimaryScrollController(
+              controller: controller,
+              child: new SizedBox(
+                height: 100.0,
+                width: 100.0,
+                child: reorderableList,
+              ),
+            ),
+          );
+        }
+
+        await tester.pumpWidget(buildWithScrollController(primary));
+        SingleChildScrollView scrollView = find.byType(SingleChildScrollView).evaluate().first.widget;
```

Comment:
Is the ".first" needed here (and below)? I think there should only be one SingleChildScrollView.

Reply:
You can use `tester.widget<SingleChildScrollView>(find.byType(SingleChildScrollView))`

Reply:
Done.